CHANGES.rst

```diff
@@ -11,6 +11,7 @@
 - Updated the default TESS quality mask to include Argabrightening, Impulsive Outlier, and Bad Calibration Exclude flags (see issue #1230). [#1336]
 - Updated tutorial notebook links [#1484]
 - Author keyword now specifies between SPOC and TESS-SPOC [#1487]
+- Modified copy() and bin() functions for FoldedLightCurve objects [#1491]
```

Comment:
1. Reword it to state the purpose (of fixing them for case FoldedLightCurve with normalized phase)
2. If `time_bin_end`,  `n_bins`, and `binsize` will no longer be supported for `FoldedLightCurve`, state it here as well given it is a breaking change.

Reply:
Added an additional comment for these deprecated keywords.